File: Lab-4/uclu_operator/src/uclu_operator/Uclu_operator.java

package uclu_operator;

import java.util.Scanner;

public class Uclu_operator {

    
    public static void main(String[] args) {
        
        Scanner in = new Scanner(System.in);
        System.out.print("Sayi:");
        int sayi = in.nextInt();
        String sonuc = (sayi % 2 == 0 ? "Sayi cifttir." : "Sayi tektir.");
        System.out.println(sonuc);
    }
    
}

File: Lab-4/uclu_operator.c
/*
	Uclu kontrol tek satirda kontrol ve sonucu yazabilmemizi saglar.
	Kontrol Durumu ? Dogru ise calisir : Dogru degil ise calisir.

*/

#include "stdio.h"
#include "stdlib.h"

int main() {

	printf("Sayi:");
	int sayi;
	scanf("%d", &sayi);
	char* sonuc;
	sonuc = (sayi % 2 == 0 ? "Sayi cifttir" : "Sayi tektir.");
	printf("%s", sonuc);

	return 0;
}